.eslintrc.json

```diff
@@ -1,7 +1,11 @@
 {
   "extends": [
     "oclif",
-    "oclif-typescript"
+    "oclif-typescript",
+    "prettier"
+  ],
```

Comment:
see prettier's recommendations for how to set it up.  The eslint **plugin** is not recommended.  We don't do it that way in salesforcecli, either.

https://prettier.io/docs/en/integrating-with-linters.html